data_model/clusters/MediaPlayback.xml

```diff
@@ -172,6 +258,43 @@ Davis, CA 95616, USA
       </mandatoryConform>
       <constraint type="desc"/>
     </attribute>
+    <attribute id="0x0007" name="ActiveAudioTrack" type="TrackStruct" default="null">
+      <access read="true" readPrivilege="view"/>
+      <quality changeOmitted="false" nullable="true" scene="false" persistence="volatile" reportable="false"/>
+      <mandatoryConform>
+        <feature name="AT"/>
+      </mandatoryConform>
+      <constraint type="desc"/>
+    </attribute>
+    <attribute id="0x0008" name="AvailableAudioTracks" type="list[TrackStruct Type]" default="null">
+      <access read="true" readPrivilege="view"/>
+      <quality changeOmitted="false" nullable="true" scene="false" persistence="volatile" reportable="false"/>
+      <mandatoryConform>
+        <feature name="AT"/>
+      </mandatoryConform>
+      <constraint type="desc"/>
+    </attribute>
+    <attribute id="0x0009" name="ActiveTextTrack" type="TrackStruct" default="null">
+      <access read="true" readPrivilege="view"/>
+      <quality changeOmitted="false" nullable="true" scene="false" persistence="volatile" reportable="false"/>
+      <mandatoryConform>
+        <feature name="TT"/>
+      </mandatoryConform>
+      <constraint type="desc"/>
+    </attribute>
+    <attribute id="0x000A" name="AvailableTextTracks" type="list[TrackStruct Type]" default="null">
+      <access read="true" readPrivilege="view"/>
+      <quality changeOmitted="false" nullable="true" scene="false" persistence="volatile" reportable="false"/>
+      <mandatoryConform>
+        <feature name="TT"/>
+      </mandatoryConform>
+      <constraint type="desc"/>
+    </attribute>
+    <attribute id="0x000B" name="AudioAdvanceMuted" type="bool" default="true">
+      <access read="true" readPrivilege="view"/>
+      <quality changeOmitted="false" scene="false" persistence="volatile" reportable="false"/>
+      <optionalConform/>
+    </attribute>
```

Comment:
```suggestion
```

This attribute was removed in PR https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7775.

instead, an AudioAdvanceUnmuted field was added to the rewind and fast forward commands, and to the new StateChanged events

Reply:
Updated.